### DISCUSSION TITLE
Configuring different keys to navigate matches after using 'open'

### DISCUSSION THREAD

**Discussion by makkus** [qutebrowser/qutebrowser#7908] [Q&A] [answered]
I'm wondering if it's possible to change the up/down arrow keys that are used to navigate between items that match the current input after using `:open'. Specifically, I'd like to use Ctrl-n/Ctrl-p. I've looked through the documentation but couldn't find the right config options, but maybe I just didn't know what terms to look for...

**Comment by The-Compiler** [comment, marked as answer]
The keyboard mode you enter after `:` is called command mode. Using `:bind --mode=command <Up>` and `:bind --mode=command <Down>` reveals that they are bound to `completion-item-focus --history prev` and `completion-item-focus --history next`, respectively ([docs](https://github.com/qutebrowser/qutebrowser/blob/main/doc/help/commands.asciidoc#completion-item-focus)).

Then, as per the [configuration docs](https://github.com/qutebrowser/qutebrowser/blob/main/doc/help/configuring.asciidoc), you can bind those keys to the same commands - e.g. when using the UI:

```
:bind --mode=command <Up> completion-item-focus --history prev
:bind --mode=command <Down> completion-item-focus --history next
```

or with `config.py`:

```python
config.bind("<Up>", "completion-item-focus --history prev", mode="command")
config.bind("<Down>", "completion-item-focus --history next", mode="command")
```

**Reply by makkus** [reply]
Thanks, perfect, that makes all sense now!
